monitoring/prometheus/tests/validate_rules.py

```diff
@@ -52,6 +66,14 @@ def load_yaml(file_name: str) -> Tuple[Dict[str, Any], str]:
     return data, errs
 
 
+def run_command(command: str):
+    c = command.split()
+    completion = subprocess.run(c, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    return (completion.returncode,
+            completion.stdout.decode('utf-8').split('\n'),
+            completion.stderr.decode('utf-8').split('\n'))
```

Comment:
If you just want to capture the stdout if successful and otherwise triggering an exception, you can enable the `check=True`, `text=True` and `capture_output=True` flags:
```suggestion
def run_command(command: str):
    c = command.split()
    return subprocess.run(c, capture_stdout=True, check=True, text=True).strip('\n')
```